Lib/test/test_typing.py

```diff
@@ -2778,6 +2778,15 @@ class HasForeignBaseClass(mod_generics_cache.A):
 
 gth = get_type_hints
 
+class ForRefExample():
+    @ann_module.dec
+    def func(self: 'ForRefExample'):
+        pass
+
+    @ann_module.dec
+    @ann_module.dec
+    def nested(self: 'ForRefExample'):
+        pass
```

Comment:
Two blank lines between top-level definitions:

```suggestion
        pass

```